packages/brite-core/src/plugins/brite-cli-plugin-webpack/__tests__/build.spec.ts

```diff
@@ -37,7 +34,11 @@ describe('build', () => {
 
     describe('production', () => {
         it('should should create node, web and css bundles', async () => {
-            await new WebpackPlugin(console, {cwd: TEST_APP_DIR, env: 'production'}).run('build');
+            await new WebpackPlugin(console, {
+                cwd: TEST_APP_DIR,
+                env: 'production',
+                basePath: 'src',
```

Comment:
What is this for? I haven't seen it before

Reply:
I had to add it in to support having a `/src` directory.

Reply:
Oh because right not apps have App.js in their folder root, this allows that nesting. Good call.